cpp/lib/src/SELinuxUtil.cc

```diff
@@ -55,7 +55,7 @@ Mode GetMode() {
 
 
 bool IsAvailable() {
-    return (ExecUtil::Which("getenforce") != "");
+    return (not ExecUtil::Which("getenforce").empty());
```

Comment:
Maybe we should cache the result here?

Reply:
ExecUtil::Which already uses a cache internally